### ISSUE TITLE
Find a way to automatically push x86_64 and aarch64 images

### ISSUE THREAD

**Issue by ericcurtin** [ericcurtin/podman-llm#14]
quay.io can only automatically build x86_64

aarch64 is important, this is tested to work on Apple Silicon/macOS with podman machine and libkrun.

we just need to figure out a way to push aarch64 images.

**Comment by cooktheryan** [comment]
Should just do multi arch builds with buildah using GitHub actions

**Comment by ericcurtin** [comment]
SGTM

**Comment by ericcurtin** [comment]
If you wanted to open a PR for that, that would be pretty neat

**Comment by ericcurtin** [comment]
We basically wanna build push all these:

https://github.com/ericcurtin/podman-llm/tree/main/container-images/podman-llm

my feeble attempt was this:

https://github.com/ericcurtin/podman-llm/blob/main/podman-build.sh

**Comment by cooktheryan** [comment]
yep. I have some time before going on leave will try to get something kicked off

**Comment by ericcurtin** [comment]
Created this:

https://quay.io/organization/ramalama

I can add you to it @cooktheryan if you let me know your username.

**Comment by rhatdan** [comment]
LGTM
